<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pause</class>
 <widget class="QDialog" name="Pause">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sono passate tot ore, prenditi una pausa per i tuoi occhi</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QTimeEdit" name="countDown">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>CONTINUE</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
